app.js

```diff
@@ -0,0 +1,5 @@
+console.log("hello People")
+
+function greet(name){
```

Comment:
Very good function display. Welldone.  I would only suggest you use an arrow function instead of a function declaration